lib/react_client.dart

```diff
@@ -1020,6 +1157,25 @@ dynamic _findDomNode(component) {
   return ReactDom.findDOMNode(component is Component ? component.jsThis : component);
 }
 
+class ReactDartContext {
+  ReactDartContext(this.Provider, this.Consumer, this._jsThis);
```

Comment:
Can we make this constructor private to decrease the public API surface area?

Reply:
I dont think i can, I construct it in createContext